demo/plt/amazonCat_13K.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+import time
+
+# Parameters
+k = 13330
+kary_tree = 16
+l = 1.0
+passes = 3
+b = 30
+output_model = "amazonCat_model"
+
+# Dataset filenames
+train_data = "amazonCat_train.vw"
+test_data = "amazonCat_test.vw"
+
+print("Vowpal Wabbit PLT example on AmazonCat-13K dataset with {} labels".format(k))
+
+# Download dataset
+if not os.path.exists(train_data):
```

Comment:
Perhaps a comment here with a link to the original source of data and how it was converted to vw format?
Specific command would be helpful for conversion would be helpful

Reply:
I've added links to a data source in each demo script + in `demo/plt/README.md` I've added note about Github repository with scripts that transform these datasets to VW format.